applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -74,7 +74,7 @@ def __init__(self, model, custom_settings):
             "residual_absolute_tolerance": 1.0e-9,
             "max_iteration": 10,
             "linear_solver_settings":{
-                "solver_type": "SuperLUSolver",
+                "solver_type": "SkylineLUFactorizationSolver",
```

Comment:
uff, your correction is correct, however this is fantastically slow. 

I would put a bit more of sofistication here, for example to verify that if a solver is not provided in the settings, than we verify if PastixSolver is availble and we use it as default, if that is not available than the Eigen direct solver, otherwise superlu ... and so on and so forth...

Reply:
1. I think the original issue is fixed in #3635 whichI created 2,5 weeks ago
2. still Riccardo made a good suggestion. Ruben I will push an improvement to this branch ok?

Reply:
@rubenzorrilla i am removing the fastpr tag. ...